src/nni_manager/training_service/local/localTrainingServiceForGPU.ts

```diff
@@ -69,9 +69,9 @@ class LocalTrainingServiceForGPU extends LocalTrainingService {
         }
     }
 
-    public cleanUp(): Promise<void> {
+    public async cleanUp(): Promise<void> {
         if (this.gpuScheduler !== undefined) {
-            this.gpuScheduler.stop();
+            await this.gpuScheduler.stop();
         }
 
```

Comment:
Also need to remove metrics data file from tmp folder in cleanup like:

remove path.join(this.gpuMetricCollectorScriptFolder, 'gpu_metrics')

Reply:
Add remove temp metrics files function.